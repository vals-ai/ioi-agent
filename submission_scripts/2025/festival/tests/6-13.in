version https://git-lfs.github.com/spec/v1
oid sha256:31bb871f953dac84e74108420e304010228c97295048aac75f1bcc3ab901822f
size 2388998
