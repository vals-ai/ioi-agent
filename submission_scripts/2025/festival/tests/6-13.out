version https://git-lfs.github.com/spec/v1
oid sha256:057b2aa6db6d4e7eb6536632e206ecaa28bbf234977a9f9b15d065ec50934959
size 2988
